audio/alac/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            mikebrady alac 96dd59d17b776a7dc94ed9b2c2b4a37177feb3c4
+version                 0.0.7
+revision                0
+categories              audio
+maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
+license                 Apache-2
+description             Apple Lossless Audio Codec and utility
+long_description        ${description}
+homepage                https://alac.macosforge.org
```

Comment:
This is Apple's ALAC project homepage which I set up for them when I moved the project from macOS forge to GitHub in 2016. Apple hasn't touched the project since then. Meanwhile, you're actually getting the code from Mike Brady's fork. I recommend removing the homepage so there's no confusion about where the software came from. Users who want to see the original homepage can click through from Mike's repository readme.
```suggestion
```